src/checkers/checkers_game.js

```diff
@@ -117,4 +114,14 @@ CheckersGame.prototype.playerDidWin = function(player) {
 };
 
 
+
+// takes in a player 
+// returns the score for the board based on the player passed in (ie. if the player 
+// has won it should return 10, if he loses should return -10) 
+CheckersGame.prototype.scoreBoard = function(player) { 
```

Comment:
This reads like scoreboard (as in the thing with numbers on it) not score as the verb. Could we use something like `evaluateState` or `evaluateBoard`